src/calls/components/CallsView.jsx

```diff
@@ -1,70 +1,138 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
+import Calendar from './Calendar';
+import { format } from 'date-fns';
 
 class CallsView extends Component {
+    state = {
+        initialItems: [],
+        items: []
+    };
 
-   state = {
-        products:
-        [{id: 1, time: "12/11/2201", duration: 51, person: "Joca", type: "Potrebna pomoc", risk: "veliki", volonter: "Stojkovic"},
-        {id: 2, time: "12/11/2051", duration: 33, person: "Ceca", type: "Potrebna pomoc", risk: "mali", volonter: "Marko"},
-        {id: 3, time: "12/11/2011", duration: 35, person: "Naca", type: "Hitan slucaj", risk: "srednji", volonter: "Ljilja"},
-        {id: 3, time: "12/11/2031", duration: 35, person: "Zaca", type: "Hitan slucaj", risk: "srednji", volonter: "Ljilja"},
-        {id: 3, time: "12/11/2001", duration: 35, person: "Kaca", type: "Hitan slucaj", risk: "srednji", volonter: "Ljilja"},
-        ]
-   };
-   renderTableData() {
-          return this.state.products.map((product, index) => {
-             const { id, time, duration, person, type, risk, volonter } = product //destructuring
-             return (
+    componentDidMount = () => {
+        let items = [
+            {
+                id: 1,
+                time: '12/11/2201',
+                duration: 51,
+                person: 'Joca',
+                type: 'Potrebna pomoc',
+                risk: 'veliki',
+                volonter: 'Stojkovic'
+            },
+            {
+                id: 2,
+                time: '12/11/2051',
+                duration: 33,
+                person: 'Ceca',
+                type: 'Potrebna pomoc',
+                risk: 'mali',
+                volonter: 'Marko'
+            },
+            {
+                id: 3,
+                time: '12/11/2011',
+                duration: 35,
+                person: 'Naca',
+                type: 'Hitan slucaj',
+                risk: 'srednji',
+                volonter: 'Ljilja'
+            },
+            {
+                id: 4,
+                time: '12/11/2031',
+                duration: 35,
+                person: 'Zaca',
+                type: 'Hitan slucaj',
+                risk: 'srednji',
+                volonter: 'Ljilja'
+            },
+            {
+                id: 5,
+                time: '24/11/2019',
+                duration: 35,
+                person: 'Kaca',
+                type: 'Hitan slucaj',
+                risk: 'srednji',
+                volonter: 'Ljilja'
+            }
+        ];
+        this.setState({ initialItems: items });
```

Comment:
Again the same issue we had before. Don't compute initial state in `componentDidMount()`.

Reply:
I missed it, but now I will fix it. 👍